### ISSUE TITLE
Resetting the View windows to their default docked positions

### ISSUE THREAD

**Issue by srk** [veusz/veusz#654]
Can this be done using the interface? I accidentally undocked some view windows, and then they refused to dock back in the original default positions. I only managed to do it on Windows by deleting the veusz.org registry key.

**Comment by jeremysanders** [comment]
No, I don't think there's any way to resetting the positions from the GUI.

**Comment by srk** [comment]
Could a button be added to the View menu to 'Reset window positions' to do this?

**Comment by PaulEdouardB** [comment]
Hi, Thanks for that amazing piece of art. Same issue here on Debian 11. Is there a config file somewhere that could help us ?
